tests/unit/libc-builtin/CMakeLists.txt

```diff
@@ -7,10 +7,15 @@ project (test-libc-builtin)
 
 add_definitions (-DRUN_ON_LINUX)
 
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-
+set (WAMR_BUILD_AOT 0)
+set (WAMR_BUILD_FAST_INTERP 0)
+set (WAMR_BUILD_INTERP 1)
+set (WAMR_BUILD_JIT 0)
+set (WAMR_BUILD_LIBC_BUILTIN 1)
 set (WAMR_BUILD_LIBC_WASI 0)
 set (WAMR_BUILD_APP_FRAMEWORK 0)
+# FIXME: if removed, LibcBuiltinTest.calloc will be failed
+set (WAMR_BUILD_MULTI_MODULE 1)
```

Comment:
By looking at the code, it shouldn't rely on multi-module; we should look into it in the future, this indeed warrants a `FIXME:`